.github/workflows/build.yaml

```diff
@@ -108,7 +108,7 @@ jobs:
           jobs: |
             tests (macos-latest)
             tests (windows-latest)
-            tests (buildjet-4vcpu-ubuntu-2204-arm)
+            test-linux
```

Comment:
You're right that "tests (buildjet-4vcpu-ubuntu-2204-arm)" should have been removed here.

But this stage already depends on `test-linux` (everything in this ci workflow does).

So the proper fix is to remove the line, thanks for figuring it out!